app/gui/src/main/scala/org/bitcoins/gui/util/GUIUtil.scala

```diff
@@ -114,10 +115,40 @@ object GUIUtil {
     }
   }
 
+  def getCopyToClipboardButton(property: StringProperty) = new Button {
+    styleClass += "icon-button" += "copy-button"
+    disable <== property.isEmpty
+    onAction = _ => {
+      setStringToClipboard(property.value)
+    }
+    tooltip = Tooltip("Copy to Clipboard.")
+    tooltip.value.setShowDelay(new javafx.util.Duration(100))
+  }
+
+  def getGreenCheck() = {
+    val img = new Image("/icons/green-check.png")
+    val imageView = new ImageView(img)
+    imageView.fitHeight = 16
+    imageView.fitWidth = 16
+    imageView
+  }
+
+  def getRedX() = {
+    val img = new Image("/icons/red-x.png")
+    val imageView = new ImageView(img)
+    imageView.fitHeight = 16
+    imageView.fitWidth = 16
+    imageView
+  }
+
   def getHSpacer() = {
     new Region { hgrow = Priority.Always }
   }
 
+  def getVSpacer() = {
```

Comment:
Should add return types to all of these functions in this file

ie `def getHSpacer(): Region`

Reply:
Marked up all the defs with return types